message tab_CharacterContainerTable 
{
	optional string tname = 1;
	repeated tab_CharacterContainer tlist = 2;
}

message tab_CharacterContainer
{
	optional int32 Id = 1;
	optional string Comment = 2;
	optional int32 Type = 3;
	optional int32 HowmanyGrids = 4;
}